src/components/searchArea/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useCallback} from 'react'
+import {FiSearch} from 'react-icons/fi'
+
+import { Container } from './styles'
+
+const SearchArea: React.FC = () => {
+  const handleWithSearch = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value
+
+    const list = document.querySelectorAll('.css-1acecyi span')
```

Comment:
Please let's avoid directly accessing the dom elements in React components. Besides that, it's using a dynamic selector, which is also bad.

Reply:
Thanks for the tips, I'll fix it